src/components/RuleCard/index.jsx

```diff
@@ -707,6 +711,15 @@ function RuleCard({ rule, onRuleDelete, ...props }) {
             )}
           </Fragment>
         )}
+        {requiresSignoff && (
+          <Fragment>
+            <Divider className={classes.divider} />
```

Comment:
We probably don't need a divider line between scheduled changes and requires signoffs.